### PR TITLE
Tournament Chuvagina

### PR DESCRIPTION
**Pull request by chuvaginaviktoriya**

### CODE REVIEW

File: 2018/Chuvagina/NewTournament/Tournament/SingleEliminationTournament.cs [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                participants.RemoveAt(index);
+            }
+
+            SaveListToBinnary("upperBracket", UpperBracketParticipants);
+            SaveListToBinnary("lowerBracket", new List<Participant>());
+        }
+
+        public SingleEliminationTournament()
+        {
+            UpperBracketParticipants = LoadListFromBinnary<Participant>("upperBracket");
```
Comment (Furmanets) [inline 2018/Chuvagina/NewTournament/Tournament/SingleEliminationTournament.cs]: Use string constants instead "upperBracket" and "lowerBracket"

File: 2018/Chuvagina/NewTournament/ConsoleTournament/ConsoleTournament.cs [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        {
+            SingleEliminationTournament tournament;
+
+            if (isFromSavedFile)
+                tournament = new SingleEliminationTournament(_inputWinner);
+            else
+            {
+                int amount = DataInput.InputAmount();
+                var participants = DataInput.InputNames(amount, _maxNameLength);
+                tournament = new SingleEliminationTournament(participants, _inputWinner);
```
Comment (Furmanets) [inline 2018/Chuvagina/NewTournament/ConsoleTournament/ConsoleTournament.cs]: SingleEliminationTournamen does't have a constructor with two parameters

File: 2018/Chuvagina/NewTournament/ConsoleTournament/ConsoleTournament.cs [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        public static void PlayDoubleElimination(bool isFromSavedFile, PrintBracket print)
+        {
+            DoubleEliminationTournament tournament;
+            if (isFromSavedFile)
+                tournament = new DoubleEliminationTournament(_inputWinner);
+            else
+            {
+                int amount = DataInput.InputAmount();
+                var participants = DataInput.InputNames(amount, _maxNameLength);
+                tournament = new DoubleEliminationTournament(participants, _inputWinner);
```
Comment (Furmanets) [inline 2018/Chuvagina/NewTournament/ConsoleTournament/ConsoleTournament.cs]: DoubleEliminationTournament does't have a constructor with two parameters

File: 2018/Chuvagina/NewTournament/Tournament/SingleEliminationTournament.cs [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+            return false;
+        }
+
+        public List<Participant> GetBracket()
+        {
+            return new List<Participant>(UpperBracketParticipants);
+        }
+
+        public static void SaveListToBinnary<T>(String fileName, List<T> serializableObjects)
```
Comment (Furmanets) [inline 2018/Chuvagina/NewTournament/Tournament/SingleEliminationTournament.cs]: I think violation of the principle of single responsibility;

Comment (chuvaginaviktoriya) [reply 2018/Chuvagina/NewTournament/Tournament/SingleEliminationTournament.cs]: Try to look at the next commit

Comment (chuvaginaviktoriya) [reply 2018/Chuvagina/NewTournament/Tournament/SingleEliminationTournament.cs]: Sorry, find out that some file weren't pushed

Comment (Furmanets) [reply 2018/Chuvagina/NewTournament/Tournament/SingleEliminationTournament.cs]: I can not find your changes, can I link?

Comment (chuvaginaviktoriya) [reply 2018/Chuvagina/NewTournament/Tournament/SingleEliminationTournament.cs]: Last commit here. Named "rest changes were added"

**Pull request merged**